typing_extensions/setup.py

```diff
@@ -9,23 +9,24 @@
                      'to install the typing package.\n')
     exit(1)
 
-version = '3.7.4.3'
+version = '3.10.0.0'
```

Comment:
I wonder if it's too early to also add to the ``classifiers`` list below this the following classifier: ``'Programming Language :: Python :: 3.10',`` while you're at it?

Reply:
Yeah I was thinking about that. Let me just run the test suite on current CPython master to be sure.

Reply:
I hope I didn't do any double work here, but FWIW, the latest CPython master runs the test with no errors on my machine.

Reply:
Thanks!